home/service/search.py

```diff
@@ -31,15 +29,40 @@ def __init__(self, form: SearchForm, page: str, items_per_page: int = 20):
         self.paginator = self._get_paginator(items_per_page)
         self.context = self._get_context()
 
+    @staticmethod
+    def _build_filter_strings(
```

Comment:
Could we rename this something like `_build_custom_property_filter`, and perhaps include a comment about how custom properties are indexed by datahub?

I was really confused why we were formatting like this until I remembered it was just for custom properties, so think this this is quite likely to confuse anyone reading the code.

Reply:
Done